src/qtt/data.py

```diff
@@ -1068,7 +1068,7 @@ def experimentFile(outputdir: str = '', tag=None, dstr=None, bname=None):
     pfile = os.path.join(outputdir, tag, basename + '.' + ext)
     return pfile
 
-
+@qtt.utilities.tools.rdeprecated(expire='1-1-2020')
```

Comment:
rdeprecated has a parameter 'txt' to help the user of this deprecated function to find the alternative or explaining the reason that this function is deprectated. We should use this parameter for deprecated functions.